TaskModules/powershell/TaskModuleSqlUtility/SqlQueryOnTargetMachines.ps1

```diff
@@ -57,6 +56,11 @@ function Execute-SqlQueryDeployment
 
         if($authscheme -eq "sqlServerAuthentication")
         {
+            if($sqlServerCredentials)
+            {
+                $sqlUsername = $sqlServerCredentials.GetNetworkCredential().username
+                $sqlPassword = $sqlServerCredentials.GetNetworkCredential().password
+            }
             $commandToRun += " -Username `"$sqlUsername`" -Password `"$sqlPassword`" "
```

Comment:
Same as above. If sqlServerCredentials is not present, you should not proceed to constructing commandToRun.